pkgs/development/tools/skaffold/default.nix

```diff
@@ -2,9 +2,9 @@
 
 buildGoPackage rec {
   name = "skaffold-${version}";
-  version = "0.16.0";
+  version = "0.18.0";
   # rev is the 0.16.0 commit, mainly for skaffold version command output
-  rev = "78e443973ee7475ee66d227431596351cf5e2caf";
+  rev = "34651689be78b2c6bcfbace5072b00b93661f895";
```

Comment:
Presumably the comment is outdated? Why is the commit hash used instead of a version tag?

Reply:
Ah, the comment should say `0.18.0` indeed… The main reason it is there is to put the hash in the `skaffold version` (https://github.com/NixOS/nixpkgs/pull/50922/files#diff-dc620cbbae6ca2d4b7cc22b6447bf85eR15)

Reply:
Ah I see. A nicer way to do this may be to use `git rev-parse HEAD` to get the current commit hash. For that you'd need to switch from `fetchFromGitHub` to `fetchgit` with the `leaveDotGit` option though to leave the git meta info intact.

If you don't want to do that, I'm fine with merging as-is. Let me know.

Reply:
@timokau Ah, wasn't sure how simple to make it work with `buildGoPackage`. I feel like we can merge this one as is and think to enhance it as a follow-up :angel: